scripts/deploy.py

```diff
@@ -268,19 +313,49 @@ def _execute_deployment():
 
         print 'Returning to oppia/ root directory.'
 
+    release_version_library_url = (
+        'https://%s-dot-%s.appspot.com/library' %
+        current_release_version, APP_NAME)
+    answer = check_errors_in_a_page(
+        release_version_library_url, 'Library page is loading correctly?')
+    if answer:
+        gcloud_adapter.switch_version(
+            APP_NAME, current_release_version)
+        print 'Successfully migrated traffic to release version!'
+    else:
+        raise Exception(
+            'Aborting version switch due to issues in library page '
+            'loading.')
+
+    if not gcloud_adapter.flush_memcache(APP_NAME):
+        print 'Memcache flushing failed. Please do it manually.'
+        memcache_url = (
+            'https://pantheon.corp.google.com/appengine/memcache?'
+            'project=%s') % APP_NAME
+        common.open_new_tab_in_browser_if_possible(memcache_url)
+
     # If this is a test server deployment and the current release version is
     # already serving, open the library page (for sanity checking) and the GAE
     # error logs.
     currently_served_version = (
         gcloud_adapter.get_currently_served_version(APP_NAME))
     if (APP_NAME == APP_NAME_OPPIATESTSERVER or 'migration' in APP_NAME) and (
             currently_served_version == current_release_version):
-        common.open_new_tab_in_browser_if_possible(
-            'https://%s.appspot.com/library' % APP_NAME_OPPIATESTSERVER)
-        common.open_new_tab_in_browser_if_possible(
+        test_server_error_logs_url = (
             'https://console.cloud.google.com/logs/viewer?'
-            'project=%s&key1=default&minLogLevel=500'
-            % APP_NAME_OPPIATESTSERVER)
+            'project=%s&key1=default&minLogLevel=500')
```

Comment:
Don't you need to interpolate the app_name?

Please check other URLs to make sure they are constructed correctly. One thing you might want to do (for neatness) is to compose all the URLs at the outset.

Reply:
Done.